time stamp
Coordinate[] objectList

float32 front_object_station
float32 front_object_velocity